client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -44,11 +45,30 @@ const StatsListCard = ( {
 		}
 	};
 
-	const outputRightItem = ( item, index ) => {
+	const toggleMobileMenu = useCallback( ( event, isVisible, key ) => {
+		event.stopPropagation();
+		event.preventDefault();
+
+		if ( isVisible ) {
+			setVisibleRightItemKey( undefined );
+		} else {
+			setVisibleRightItemKey( key );
+		}
+	}, [] );
```

Comment:
Doesn't this array need `isVisible`?

Reply:
It doesn't need. `isVisible` only exists inside the callback function. You only pass to the array of dependencies values that belong to the component scope, like state variables or props.

Reply:
In fact, this what you got if you try to pass `isVisible` as a dep to useCallback

<img width="992" alt="image" src="https://user-images.githubusercontent.com/6406071/210465941-f4ca8819-4570-481b-8133-005b67f58d51.png">